entry_types/scrolled/package/spec/frontend/MediaPlayer-spec.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect'
+import 'support/mediaElementStub';
+
+import {renderInEntry} from 'support';
+
+import {MediaPlayer} from 'frontend/MediaPlayer';
+
+describe('MediaPlayer', () => {
+  let audioSources =[ 
+    {type: 'audio/ogg', src: 'http://example.com/example.ogg'},
+    {type: 'audio/m4a', src: 'http://example.com/example.m4a'},
+    {type: 'audio/mp3', src: 'http://example.com/example.ogg'}
+  ];
+
+  it('do not renders audio tag without sources', () => {
+    const {container} = renderInEntry(<MediaPlayer type={'audio'} />, {
+      container: document.createElement('div')
+    });
+    expect(container.querySelector('audio')).toBeNull();
+  });
+
+  it('renders audio tag for audio sources', () => {
+    const {container} = renderInEntry(<MediaPlayer type={'audio'} sources={audioSources} />, {
+      container: document.createElement('div')
+    });
+    
+    expect(container.querySelector('audio')).toBeDefined();
+  });
+
```

Comment:
The component has more logic then what is tested here. Especially the releasePlayer part seems interesting. I'd suggest using [react-testing-library's unmount helper](https://testing-library.com/docs/react-testing-library/api#unmount) and [spy on](https://jestjs.io/docs/en/jest-object#jestspyonobject-methodname) the releasePlayer method.

Reply:
added one more spec 'releases media player on component unmount'